test/cypress/integration/paying-for-college/disclosures/disclosures-helpers.cy.js

```diff
@@ -7,16 +7,19 @@ export class DynamicDisclosures {
     return cy.get('#estimated-years-attending');
   }
 
-  confirmVerification() {
-    cy.get('a[href="#info-right"]').click({ force: true });
+  confirmVerification(forceParam) {
+    if ( typeof forceParam === 'undefined' ) forceParam = true;
```

Comment:
Can you do `confirmVerification(forceParam = true) {` ?

Here and below.

Reply:
We can do that, I think. Does forcing those help speed up the testing? I don't see a reason not to force them.